bin/docker-image-tool.sh

```diff
@@ -41,6 +41,18 @@ function image_ref {
   echo "$image"
 }
 
+function docker_push {
+  local image_name="$1"
+  if [ ! -z $(docker images -q "$(image_ref ${image_name})") ]; then
+    docker push "$(image_ref ${image_name})"
+    if [ $? -ne 0 ]; then
+      error "Failed to push $image_name Docker image."
```

Comment:
Script should really fail i.e. `exit` if a push fails

Reply:
Same comment as above. Script does exit with non-zero exit code when push fails.